src/plugins/vue.js

```diff
@@ -0,0 +1,64 @@
+import { escapeHTML } from "../lib/utils";
+
+function hasValueOrEmptyAttribute(value) {
+  return Boolean(value || value === "");
+}
+
+const NO_HIGHLIGHT = "no-highlight";
```

Comment:
Hmm, looks like this won't work in case of `hljs.configure({noHighlightRe: 'something'})`...
(and the same is true for `blockLanguage` function in `highligh.js`)

Reply:
Damn. :-) Grrr.

Reply:
Probably it's safe to always check for `/^(no-?highlight)$/i` in `shouldNotHighlight`, and additionally check for the provided regex?

Reply:
HLJS doesn't expose it's internal settings though. :-)

Reply:
Actually we still hard-code it for when an explicit language "fails" to load.  So this has the exact same [fixed] behavior as the main library. So I'm calling this good enough for now. :-)

```js
      if (!language) {
        console.warn(LANGUAGE_NOT_FOUND.replace("{}", match[1]));
        console.warn("Falling back to no-highlight mode for this block.", block);
      }
      return language ? match[1] : 'no-highlight';
    }
```

Reply:
> Actually we still hard-code it for when an explicit language "fails" to load.

Yes, that's exactly what I mentioned in my first message (: 
So what do you think about changing `shouldNotHighlight` to always check for this hardcoded value? (in addition to whatever may be passed in `options`)

Reply:
I see two choices:

- Drop the configurability completely and just have a static value/regex.
- Improve the configurability to make this "no-highlight" value something you could provide via options

This configurability was only added preemptively when we changed the meaning of "plain" and "text" - to allow people to easily get the "old" behavior.

Really this behavior is just a way to "alert" the user visually of broken highlighting without making a big fuss about it (break the styles a little bit).  We could simply deprecate that behavior and when there is an error apply no css classes at all, simply let the user read the console.log if they want to know what went wrong?

Reply:
I think I'd rather just remove the hard-coded string... so the only way any type of "no-highlight" gets on there is if the user added it themself... the library would never do it.

Reply:
> I think I'd rather just remove the hard-coded string... so the only way any type of "no-highlight" gets on there is if the user added it themself... the library would never do it.

Probably it's the right way... but you have to do something in the language-not-found case.
Just wanted to say that this issue shouldn't stop the PR. Looks like It should be addressed separately.